sidebars.js

```diff
@@ -100,6 +100,37 @@ const sidebars = {
     },
     'ecosystem/index',
     'extensions/index',
+    {
+      type: 'category',
+      label: 'Aries Framework JavaScript Extensions',
```

Comment:
Can we add a link on the ecosystems page in the ext section that link towards the documentation for the ext packages?